src/modules/test/recipes/Backup/SecurityBaselineTests_full.json

```diff
@@ -0,0 +1,1957 @@
+[
```

Comment:
What is the difference between this backup/full test and the other test recipe? Can they be a single test?

Reply:
At some point in the future when the entire Security Baseline will finished as the SSH part is, there will be one test. For now, only the SSH part can remain in the official test (also have to remove remaining audits here)